src/components/Home/Meetings/views/Recurring.tsx

```diff
@@ -1,20 +1,29 @@
+import { useState, useCallback } from "react";
 import get from "lodash/get";
```

Comment:
we're still not using the singular libraries, please change to lodash.get

Reply:
I caught the fail because I'm using a lot of functions from `lodash`. And there is a conflict in the names of the functions inside the different modules and Vite doesn't know what to do with it.

![image](https://user-images.githubusercontent.com/11043120/231726727-c874b4d3-669a-495f-a5ea-bb309e799708.png)

Reply:
dont use uppercase, use always like lodash.isempty instead of lodash.isEmpty
each lodash module uses a single function from the repo, which you can see https://github.com/lodash/lodash

Reply:
There are two way:
1. We leave a package of `lodash`
2. If you have a fix, I will replace it (_I don't want to waste time on it, it's not rational_)

Reply:
replaced to modules